core-libs/setup/ssr/optimized-engine/ssr-optimization-options.ts

```diff
@@ -136,6 +137,39 @@ export interface SsrOptimizationOptions {
    * By default, the DefaultExpressServerLogger is used.
    */
   logger?: ExpressServerLogger;
+
+  /**
+   * When caching is enabled, this function tell whether the given rendering result
+   * (html or error) should be cached.
+   *
+   * By default, all html rendering results are cached. By default, also all errors are cached
+   * unless the separate option `avoidCachingErrors` is enabled.
+   */
+  shouldCacheRenderingResult?: ({
+    options,
+    entry,
+  }: {
+    options: SsrOptimizationOptions;
+    entry: Pick<RenderingEntry, 'err' | 'html'>;
+  }) => boolean;
+
+  /**
+   * Toggles providing granular adaptation to breaking changes in OptimizedSsrEngine.
+   * They are temporary and will be removed in the future.
+   * Each toggle has its own lifespan.
+   */
+  featureToggles?: {
```

Comment:
[MEDIUM]
`featureToggles` vs `ssrFeatureToggles`?
Why asking: whenever I mention it, it can be so eaisly confused with the main feature toggles (which we will have hundreads soon). IMHO we should be clear - by the name - that those are not the same toggles as the main ones. Therefore I'm suggesting a prefixed name, to distinguish it. What do you think about it?